.gitignore

```diff
@@ -18,9 +18,10 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
```

Comment:
it's really need it, i just checked it and there is no files added without it

Reply:
you have to show me why u added it

Reply:
No problem amigo xd i can show u this shit xD but webstorm accepted this file as invisible after add this entry